### DISCUSSION TITLE
Help regarding key file authenication

### DISCUSSION THREAD

**Discussion by ghost** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#676] [Q&A]
Hello,
I have followed all the steps to set up the assistant, and ive uploaded the OAuth2 Credentials json file to the app, but it still says.
![image](https://user-images.githubusercontent.com/81710515/158675549-558327eb-3619-43ec-9b3c-fe62a99836e6.png)

I checked the file name and it is in that format

**Comment by ghost** [comment]
> Hello,
> I have followed all the steps to set up the assistant, and ive uploaded the OAuth2 Credentials json file to the app, but it still says.
> ![image](https://user-images.githubusercontent.com/81710515/158675549-558327eb-3619-43ec-9b3c-fe62a99836e6.png)
> 
> I checked the file name and it is in that format

Fixed, I followed steps in the issues tab

**Comment by elisha1242** [comment]
i have the same problem, dont know what to do.. someone?

**Comment by JustPita** [comment]
same problem

**Comment by im-coder-lg** [comment]
People, the solution is in #671. Basically check the comments, the main thing to do is get a different JSON file for the same project, which has the correct keys in it.
